Sources/Datadog/Utils/SwiftExtensions.swift

```diff
@@ -6,10 +6,52 @@
 
 import Foundation
 
+// MARK: - Optional
+
 extension Optional {
     func ifNotNil(_ closure: (Wrapped) throws -> Void) rethrows {
         if case .some(let unwrappedValue) = self {
             try closure(unwrappedValue)
         }
     }
 }
+
+// MARK: - Date
+
+extension Date {
+    // NOTE: RUMM-182 counterpart of currentTimeMillis in Java
+    // https://docs.oracle.com/javase/7/docs/api/java/lang/System.html#currentTimeMillis()
+    var currentTimeMillis: UInt {
```

Comment:
because it overflows. so we should explicitly use `UInt64`?

Reply:
> because it overflows. so we should explicitly use UInt64?

Yes, it's the only way to represent current timestamp on `32-bit` devices. This will be less performant as not optimised in CPU, but we have no other way.